aten/src/TH/THMemoryFile.cpp

```diff
@@ -51,23 +51,25 @@ static int8_t *THMemoryFile_strnextspace(int8_t *str_, int8_t *c_)
 static void THMemoryFile_grow(THMemoryFile *self, ssize_t size)
 {
   ssize_t missingSpace;
+  uint64_t nbytes = self->storage->nbytes();
 
   if(size <= self->size)
     return;
   else
   {
-    if(size < self->storage->numel()) /* note the "<" and not "<=" */
+    if (size < nbytes) /* note the "<" and not "<=" */
```

Comment:
Are you sure this is right?

Reply:
It looks like you silently changed the semantics of the `size` input to be bytes now? (Or was it bytes already?)

Reply:
It seemed like THMemoryFile is always going to be filled with 1-byte characters, so numel would be equal to nbytes. But I could be wrong, let me double check that.

Reply:
It uses `THCharStorage` so it's explicitly working in bytes.